{u'description': [u"Voyageur's random ebuilds stuff"],
 u'feeds': [u'https://cafarelli.fr/cgi-bin/cgit.cgi/voyageur-overlay/atom'],
 u'homepage': u'https://cafarelli.fr/cgi-bin/cgit.cgi/voyageur-overlay/',
 u'irc': None,
 u'name': u'voyageur',
 u'official': True,
 u'owner_email': u'voyageur@gentoo.org',
 u'owner_name': None,
 u'priority': 50,
 u'quality': u'experimental',
 u'sources': [(u'https://cafarelli.fr/git/voyageur-overlay/', u'Git', u'')],
 u'status': u'official',
 u'supported': True}
$ pmaint sync voyageur
WARNING:pkgcore:repository at '/home/mgorny/repos/steveeJ' has unsupported profile format(s): 'profile-set'
Already up-to-date.
*** syncing voyageur
*** synced voyageur
 * Sync succeeded
$ pmaint regen -t 2 voyageur
WARNING:pkgcore:repository at '/home/mgorny/repos/steveeJ' has unsupported profile format(s): 'profile-set'
WARNING:pkgcore:repository at '/home/mgorny/repos/steveeJ' has unsupported profile format(s): 'profile-set'
!!! caught exception bytes_ro_StringIO instance has no attribute '__len__' while processing ebuild src: app-editors/texworks-0.4.6
!!! caught exception bytes_ro_StringIO instance has no attribute '__len__' while processing ebuild src: app-admin/aws-cli-1.7.36
caught exception bytes_ro_StringIO instance has no attribute '__len__' while processing ebuild src: app-admin/vca-cli-11_rc1
!!! caught exception bytes_ro_StringIO instance has no attribute '__len__' while processing ebuild src: www-apps/gateone-1.1
!!! caught exception bytes_ro_StringIO instance has no attribute '__len__' while processing ebuild src: www-apps/gateone-9999
!!! caught exception bytes_ro_StringIO instance has no attribute '__len__' while processing ebuild src: games-board/pysolfc-9999
!!! caught exception bytes_ro_StringIO instance has no attribute '__len__' while processing ebuild src: dev-python/botocore-1.0.1
!!! caught exception bytes_ro_StringIO instance has no attribute '__len__' while processing ebuild src: dev-python/iptools-0.6.1
!!! !!! caught exception bytes_ro_StringIO instance has no attribute '__len__' while processing ebuild src: net-misc/dagr-0.60
caught exception bytes_ro_StringIO instance has no attribute '__len__' while processing ebuild src: dev-python/jmespath-0.7.1
!!! caught exception bytes_ro_StringIO instance has no attribute '__len__' while processing ebuild src: dev-python/robobrowser-0.5.1
caught exception bytes_ro_StringIO instance has no attribute '__len__' while processing ebuild src: dev-python/pyvcloud-12
!!! caught exception bytes_ro_StringIO instance has no attribute '__len__' while processing ebuild src: app-backup/rdiff-backup-1.3.3
!!! caught exception bytes_ro_StringIO instance has no attribute '__len__' while processing ebuild src: dev-python/robobrowser-0.5.2
!!! caught exception bytes_ro_StringIO instance has no attribute '__len__' while processing ebuild src: net-misc/dagr-9999
 * Cache regenerated successfully
